_docs/automation-time-apis.md

```diff
@@ -121,6 +121,165 @@ pub enum Error {
+
+#### Call
+```rust
+fn calculate_optimal_autostaking(
+    /// The amount that will be staked.
+    principal: i128,
+    /// The collator to which the principal will be staked.
+    collator: AccountId,
+)
+```
+
+* This RPC will return a JSON object with a time period and an APY. The time period represents the optimal staking duration and the APY is the projected staking return.
+
+#### Errors
+* **Unable to calculate optimal autostaking**: The optimal autostaking calculation went wrong.
+* **collator does not exist**: The collator provided is not a real collator.
+* **could not calculate fee**: The fee calculation went wrong.
+
+#### Example
+Sample Request
+```cURL
+curl --location --request POST 'http://rpc.turing-staging.oak.tech' \
+--header 'Content-Type: application/json' \
+--data-raw '{"id":1, "jsonrpc":"2.0", "method": "automationTime_calculateOptimalAutostaking", "params": [1000000000000, "6AwtFW6sYcQ8RcuAJeXdDKuFtUVXj4xW57ghjYQ5xyciT1yd"]}'
+```
+
+Sample Response
+```JSON
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "period": 63,
+        "apy": 0.08711145708327778
+    },
+    "id": 1
+}
+```
+
+### RPC: Get Task IDs for Auto-compounding staking delegation tasks.
+This API gets the task IDs for auto-compounded stake delegation tasks. Auto-compounding staking delegation tasks are unique to an wallet address and collator pair. This RPC collects all of the task IDs for existing auto-compounding staking delegation tasks for each unique wallet address to collator pair, given a single wallet address. The auto-compounding task must exist in order for this RPC to return a task ID. The delegator must also have delegated to a collator first in order for the auto-compounding task to be included. 
+
+#### Call
+```rust
+fn get_auto_compound_delegated_stake_task_ids(
+    /// The account ID of the wallet of the delegator.
+    account_id: AccountId
+)
+```
+
+* This RPC will return the task IDs of the auto-compounding staking delegation task for each collator that this wallet has staked with.
+
+#### Errors
+* **Unable to get AutoCompoundDelegatedStakeTask ids**: Retrieval of Task IDs went wrong.
+
+#### Example
+Sample Request
+```cURL
+curl --location --request POST 'http://rpc.turing-staging.oak.tech' \
+--header 'Content-Type: application/json' \
+--data-raw '{"id":1, "jsonrpc":"2.0", "method": "automationTime_getAutoCompoundDelegatedStakeTaskIds", "params": ["68vqVx27xVYeCkqJTQnyXrcMCaKADUa7Rywn9TSrUZyp4NGP"]}'
+```
+
+Sample Response
+```JSON
+{
+    "jsonrpc": "2.0",
+    "result": [
+        "0xceff6bb4a24529cbd4707cafd9ee158774ecbdc31e3479e1452a259df91f2c50",
+        "0xb222b3501b23c6482894b6051d7fcbae7de3da9c8a8dc9e54971dbaff129b028"
+    ],
+    "id": 1
+}
+```
+
 ## Coming soon
```

Comment:
Actually we should clean this up / remove this heading since it's live

Reply:
ahh, just realized, this isn't live in the way that we think it is. it's live through the schedule xcmp task, where you need another parachain to initiate the action. i'll write that in as an experimental section